src/mame/drivers/midvunit.cpp

```diff
@@ -2,28 +2,30 @@
 // copyright-holders:Aaron Giles
 /*************************************************************************
 
-    Driver for Midway V-Unit games
+    Williams/Midway V-Unit system
 
-    driver by Aaron Giles
+    driver originally by Aaron Giles
 
     Games supported:
         * Cruis'n USA (1994)        [3 sets]
         * Cruis'n World (1996)      [7 sets]
         * War Gods (1996)           [3 sets]
         * Off Road Challenge (1997) [5 sets]
 
-    Known bugs:
-        * textures for automatic/manual selection get overwritten in Cruis'n World
-        * rendering needs to be looked at a little more closely to fix some holes
+    Todo:
+        * Cruis'n World's neutral gear texture corruption
+        * Both games will eventually freeze in attract
         * in Cruis'n World attract mode, right side of sky looks like it has wrapped
-        * Off Road Challenge has polygon sorting issues, among other problems
-        * Issues for the Wargods sets:
-           Sound D/RAM      ERROR EE (during boot diag)
-           Listen for Tone  ERROR E1 (during boot diag)
-           All sets report as Game Type: 452 (12/11/1995) [which is wrong for newer sets]
+        * Cruis'n USA's ending plays at double speed (protection related?)
+        * Get the linking feature to actually work (controlled by a mix of security pals and bus transceivers?)
```

Comment:
These kinds of uninformed guesses are not helpful.  If you don't actually have evidence that something is a form of protection, don't blame it on protection.  The linking isn't controlled by the PALs and transceivers - they're just part of the physical interface between boards.

Reply:
May as well give permission to delete those uninformed guesses. Phil Bennett mentioned the Midway V Unit communications were done through a parallel connection judging by the schematics from a Cruis'n USA manual. What also made me think that PAL located at U38 was responsible for comms was because a Cruis'n USA link kit required replacing that PAL otherwise both pcbs won't communicate.